dashboard/app/helpers/profanity_helper.rb

```diff
@@ -0,0 +1,14 @@
+require 'digest/md5'
+
+module ProfanityHelper
+  # Uses ProfanityFilter to find any profanities in the given text + locale. Caches the response.
+  # @param [String] text
+  # @param [String] locale
+  # returns [Array<String>|nil] Array of profane words, if any are found.
+  def find_profanities(text, locale)
+    return nil if text.nil_or_empty?
+    # Hash text in cache_key to avoid long cache keys.
+    cache_key = "profanity/#{locale}/#{Digest::MD5.hexdigest(text)}"
```

Comment:
Why MD5? I think best practice is not to use that anymore. Maybe doesn't matter since we don't care much about security, just compressing size?

Reply:
yeah, we don't need this to be secure -- it's just for compressing size and making sure we don't have any spaces or invalid characters in the cache key